src/org/apache/xerces/xni/QName.java

```diff
@@ -71,12 +72,21 @@ public QName() {
         clear();
     } // <init>()
 
-    /** Constructs a QName with the specified values. */
+    /**
+     * Constructs a QName with the specified values.
+     * @param prefix    the qname prefix. (e.g. "a")
+     * @param localpart the qname localpart. (e.g. "foo")
+     * @param rawname   the qname rawname. (e.g. "a:foo")
+     * @param uri       the URI binding. (e.g. "http://foo.com/mybinding")
+     */
     public QName(String prefix, String localpart, String rawname, String uri) {
         setValues(prefix, localpart, rawname, uri);
     } // <init>(String,String,String,String)
 
-    /** Constructs a copy of the specified QName. */
+    /**
+     * Constructs a copy of the specified QName.
+     * @param qname
```

Comment:
please include a description if you're adding a param tag. The argument name alone isn't needed.